vmm/src/vstate.rs

```diff
@@ -262,7 +262,14 @@ impl Vcpu {
         )
         .map_err(Error::CpuId)?;
 
-        filter_cpuid(&mut self.cpuid, &cpuid_vm_spec).map_err(Error::CpuId)?;
+        filter_cpuid(&mut self.cpuid, &cpuid_vm_spec).map_err(|e| {
+            METRICS.vcpu.filter_cpuid.inc();
+            error!(
+                "Failure in configuring CPUID for vcpu {:?}: {:?}",
```

Comment:
`self.id` is an `u8` and can be formatted with the `{}` operator:
```suggestion
                "Failure in configuring CPUID for vcpu {}: {:?}",
```

Reply:
Ok, I will fix this